library/src/main/java/com/shawnlin/numberpicker/NumberPicker.java

```diff
@@ -241,6 +246,11 @@ public static final Formatter getTwoDigitFormatter() {
      */
     private int mTextColor = DEFAULT_TEXT_COLOR;
 
+    /**
+     * The color of the highlighted text.
+     */
+    private int mIntervalTextColor = DEFAULT_TEXT_COLOR;
```

Comment:
It seems that `mIntervalTextColor` is the same as `mSelectedTextColor`.
Could you please give me more information about the details and screenshots?